cmd/utils.go

```diff
@@ -60,21 +69,26 @@ func getDepInfo() (map[string][]string, []string, string) {
 			depGraph[words[0]] = append(depGraph[words[0]], words[1])
 		}
 
+		isMainModule := false
```

Comment:
Is there a reason we need this extra variable instead of the previous approach?

Reply:
with the original code, this was never called for the first dependency `deps = append(deps, words[1])` and I think that's wrong, right? If that was leaf dependency, it would not get to the result list

Reply:
Ah yes! I totally missed that 😅